OpenSubtitlesHandler/Models/ApiResponse.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace OpenSubtitlesHandler.Models
+{
+    public class ApiResponse<T>
+    {
+        public HttpStatusCode Code { get; }
+        public string Body { get; } = string.Empty;
+        public int Remaining { get; }
+        public int Reset { get; }
+        public Dictionary<string, string> Headers { get; }
+        public T Data { get; }
+
+        public ApiResponse((T response, (int remaining, int reset) limits, Dictionary<string, string> headers, HttpStatusCode statusCode) input)
```

Comment:
Why use a tuple of parameters instead of just passing the parameters?

Reply:
This was inherited from the previous/current version, I can change it if its a problem though

Reply:
changed to params in 7327c8a2b6b17795f814c6ea88f356c9ad783a45